File: aoj/volume100/csharp/A10022.cs
using System;
using System.Linq;

class AOJ
{
	static void Main()
	{
		var pattern = Console.ReadLine();
		var cnt = 0;
		while (true)
		{
			var line = Console.ReadLine();
			if (line == "END_OF_TEXT")
				break;

			cnt += line
				.Split()
				.Select(s => s.ToLower())
				.Where(s => s == pattern)
				.Count();
		}
		Console.WriteLine(cnt);
	}
}

File: aoj/volume100/scala/A10022.scala
object Main {
  def main(args: Array[String]) {
    val w = scala.io.StdIn.readLine.toLowerCase
    var count = 0
    for {
      line <- Iterator.continually(scala.io.StdIn.readLine).takeWhile(_ != "END_OF_TEXT")
      word <- line split " "
      if w == word.toLowerCase 
    } count += 1
    println(count)
  }
}
